### ISSUE TITLE
Installation error on tailwindcss build with `cargo install`

### ISSUE THREAD

**Issue by haruka-7** [axodotdev/oranda#631]
Hi !

I have an error when I try to install Oranda, by doing `cargo install oranda`, regarding the Tailwindcss build.

What information can I provide to be more helpfull ?

```
error: failed to run custom build command for `oranda v0.3.1`

Caused by:
  process didn't exit successfully: `/tmp/cargo-installSOq29b/dist/build/oranda-65165f5ef0192407/build-script-build` (exit status: 101)
  --- stderr
  Specified input file /home/hidrile/.cargo/registry/src/index.crates.io-6f17d22bba15001f/oranda-generate-css-0.3.1/../oranda-css/css/main.css does not exist.
  thread 'main' panicked at 'Tailwind failed to compile CSS!', /home/hidrile/.cargo/registry/src/index.crates.io-6f17d22bba15001f/oranda-generate-css-0.3.1/src/lib.rs:94:10
  note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
warning: build failed, waiting for other jobs to finish...
error: failed to compile `oranda v0.3.1`, intermediate artifacts can be found at `/tmp/cargo-installSOq29b`
```

**Comment by manyinsects** [comment]
This seems to be broken right now and I'm not sure why! The `cargo install` used to work on this exact same version (unless my memory is failing me). I recommend either using the install script on https://opensource.axo.dev/oranda (it installs to the same place as `cargo install` does), or to install the latest prerelease (`cargo install oranda@0.4.0-prerelease.2`), which doesn't have this issue.

**Comment by haruka-7** [comment]
The install script works indeed. Thanks!

**Comment by manyinsects** [comment]
(also just to add, we're investigating this issue! it's definitely weird because it _used_ to work. pinning this issue for other people who come across it)

**Comment by manyinsects** [comment]
Oh, forgot to say this is fixed with `0.4.0`!
